packages/locales/extractReqLocale.js

```diff
@@ -14,7 +14,7 @@ const ACCEPT_LOCALES = ['ru', 'en']
  * @returns {string | null}
  */
 const extractReqLocale = (req) => {
-    if (!req) return null
+    if (!req) return undefined
```

Comment:
That's a breaking change. Many apps uses it, and can compare `if (locale !== null)`, Also see return type above: `string | null`, undefined return us not declared